cluster/addons/fluentd-elasticsearch/fluentd-es-image/Dockerfile

```diff
@@ -19,8 +19,10 @@
 # Note that fluentd is run with root permssion to allow access to
 # log files with root only access under /var/log/containers/*
 
-FROM gcr.io/google-containers/debian-base-amd64:0.1
+FROM debian:stretch-slim
 
+COPY clean-apt /usr/bin
```

Comment:
sorry, unclear, I meant the docker image this is a part of.  

this script `clean-apt` is being executed in the current HEAD version of this Dockerfile, I do not think this could have been successfully built as this file wasn't existing.